python3/ltk/actions/init_action.py

```diff
@@ -462,10 +459,8 @@ def set_workflow(self, community_id, project_id):
             # End Python 3
             try:
                 if choice == '':
-                    for x in workflow_info:
-                        if x == workflow_id:
-                            workflow_id, workflow_name = x, 'Project Default'
-                    logger.info('\nSelected "{0}" {1}.\n'.format(workflow_name, 'workflow'))
+                    workflow_id, workflow_name = 'Project Default', 'Project Default'
```

Comment:
Maybe I'm not understanding correctly but it looks like if they select to use the "Project Default" workflow, we literally send the workflow_id of `Project Default` to TMS. I'm not sure TMS is able to handle that value, instead we should either send an empty workflow id, i.e `''` or not send the workflow parameter at all.

Reply:
When the `Project Default` is chosen it just saves the value to the file config. When something other than `Project Default` is chosen then it updates the project with that workflow so that it becomes the new `Project Default` and the id is saved to config. The id stored in config is never referenced again in the code from what I could tell.